libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilotImpl.kt

```diff
@@ -312,6 +312,7 @@ internal class MapboxCopilotImpl(
         val sessionId = generateSessionId(copilotMetadata, owner)
         return AttachmentMetadata(
             name = filename,
+            created = startedAt,
```

Comment:
Why do we need to create `AttachmentMetadata` in two different places? In theory, it should be created once right before the upload

Reply:
That's a great question. Problem is that we cannot pass objects to the `HistoryUploadWorker` so we need to recreate it. Decided to recreate it so we don't need to pass primitives over (trying to avoid [Long Parameter List](https://refactoring.guru/smells/long-parameter-list) and [Primitive Obsession](https://refactoring.guru/smells/primitive-obsession) code smells), it seems that's more OOP and a better approach overall.